source/agora/test/NetworkManager.d

```diff
@@ -169,16 +197,28 @@ unittest
         last_txs = txs;
     }
 
-    // the validator node has 10 blocks, but bad node pretends to have 20
-    assert(node_validator.getBlockHeight() == 10, node_validator.getBlockHeight().to!string);
-    assert(node_bad.getBlockHeight() == 20);
+    // create genesis block
+    last_txs = genesisSpendable().map!(txb => txb.sign()).array();
+    last_txs.each!(tx => node_validator.putTransaction(tx));
+    network.expectBlock([node_validator], Height(1), 4.seconds);
```

Comment:
The foreach loop before this code creates 9 blocks, so how come this test here passes?

Reply:
It seems like some of my changes were not applied during the rebase.
Will check again.